src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/CapabilityFilteredBrowse.feature

```diff
@@ -1,33 +0,0 @@
-﻿Feature: Capability-filtered browse
```

Comment:
where is this covered if you're removing it?

Reply:
Part of next story covers this